src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -868,7 +868,42 @@ void MFEMSidreDataCollection::Load(const std::string& path,
     std::string suffixedPath = endsWith(path, ".root") ? path : path + ".root";
 
     IOManager reader(m_comm);
-    reader.read(m_bp_grp->getDataStore()->getRoot(), suffixedPath);
+    // Get the paths in which the global and domain groups are stored
+    // This allows for a datastire structure other than the one used when this
+    // class owns the DataStore
+    Group* global_grp = m_bp_index_grp->getParent()->getParent();
+    Group* domain_grp = m_bp_grp->getParent();
```

Comment:
Can either of these ever be `null` ?

Reply:
I think we should be safe here - my interpretation of the `getParent` docs is that it will never return `nullptr`:

https://github.com/LLNL/axom/blob/8478a6f3552f68af66bb11ae18a0d516d2d35305/src/axom/sidre/core/Group.hpp#L185-L188

Reply:
Sorry for not being clearer, I was referring to `m_bp_index_grp` and `m_bp_grp` -- are they always non-null?

Reply:
I'll add a check for this